Marlin/src/pins/stm32f4/pins_MKS_ROBIN_NANO_V3.h

```diff
@@ -178,9 +178,11 @@
 //
 // Misc. Functions
 //
-#define MT_DET_1_PIN                        PA4
-#define MT_DET_2_PIN                        PE6
-#define MT_DET_PIN_INVERTING               false  // LVGL UI filament RUNOUT PIN STATE
+#if HAS_TFT_LVGL_UI
+  #define MT_DET_1_PIN                      PA4   // MT_DET
+  #define MT_DET_2_PIN                      PE6   // FALA_CRTL
```

Comment:
Is this the mark on the board? Seems like it should be "FILA_CTRL."

Reply:
> Is this the mark on the board? Seems like it should be "FILA_CTRL."

I checked mine & it isn't marked as such. The silkscreen says `PW_DET`, `MT_DET1`, `MT_DET2`, etc. on top with the pin numbers on the bottom. This matches up with the TOP/BOTTOM pdfs in [MKS' Robin Nano V3 repo](https://github.com/makerbase-mks/MKS-Robin-Nano-V3.X/tree/main/hardware/MKS%20Robin%20Nano%20V3.0_004) as well.